src/Cassandra/Mapping/Mapper.cs

```diff
@@ -24,6 +24,10 @@
 using Cassandra.SessionManagement;
 using Cassandra.Tasks;
 
+#if !NETSTANDARD2_1_OR_GREATER
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
same scenario as the previous one, I prefer to have methods separate so it's more clear when things are async and when they aren't

Reply:
For this case, since they are public methods, I didn't add new methods but made their implementations differ depending on the target framework.

Reply:
Sure, that's what I went for. It's a little more tricky because the return statement should also be changed (`return` VS `return Task.FromResult`)